aces/Loris
# [new_profile] Fix datepicker default values.

## Brief summary of changes
Fix the datapicker default values for the "Date of Birth" and "Date of Birth Confirm" fields. Updates the testPlan.

- [x ] Have you updated related documentation?

#### Testing instructions (if applicable)

1. Go to the `Configuration` module and change the **"Minimum candidate age**" value.
2. Now go to the `new_profile` module and make sure **"Date of Birth"** and **"Date of Birth Confirm"** fields
   properly reflects the changes.
3. Go again to the `Configuration `and now delete the value in the **"Minimum candidate age"** field, and save.
4.  Now go to the new_profile module again make sure **"Date of Birth"** and **"Date of Birth Confirm"** fields have as the maximum possible value to choose the current day.


#### Link(s) to related issue(s)

* Resolves #  [(Reference the issue this fixes, if any.)](https://github.com/aces/Loris/issues/7996)
